src/muse/readers/csv.py

```diff
@@ -612,19 +636,31 @@ def process_technologies(
         )
     outs = outs.drop_vars("flexible_outputs")
 
-    # Interpolate inputs/outputs if needed
-    if "year" in technodata.dims and len(technodata.year) > 1:
-        outs = outs.interp(year=technodata.year)
-        ins = ins.interp(year=technodata.year)
+    # Collect all years from the time framework and data files
+    time_framework = list(
```

Comment:
This list of years is unsorted. Is that OK for the `interpolate_technodata` function?